lab_1/start.py

```diff
@@ -2,17 +2,41 @@
 Concordance implementation starter
 """
 
-from main import read_from_file
 import os
+import main

```

Comment:
Usually, there are imports like:
`from main import func1, func2, func3 ...` unless you want to use every function from the `main` module.